tests/test_flatAndIlluminationCorrection.py

```diff
@@ -97,29 +102,27 @@ def testIllum3(self):
         self.doIllum(scaling=3.7)
 
     def testGainAndReadnoise(self):
-        import lsst.afw.image as afwImage
-        from lsst.afw.cameraGeom.testUtils import DetectorWrapper
-        from lsst.ip.isr import IsrTask
 
         isrTask = IsrTask()
 
         detector = DetectorWrapper().detector
         raw = afwImage.ExposureF(detector.getBBox())
 
         level = 10
-        readNoise = 1
+        readNoise = 1.5
         raw.image.set(level)
 
         amp = detector[0]
         amp.setReadNoise(readNoise)
 
-        for gain in [-1, 0, 0.1, 1]:
+        for gain in [-1, 0, 0.1, 1, float("NaN")]:
```

Comment:
Might be better to use `math.nan` (or use numpy not math)

Reply:
Fixed.  I followed the other examples in this package that do this, but you are right.